packages/api/src/controllers/stream.ts

```diff
@@ -1085,8 +1097,8 @@ function publishDelayedRecordingReadyHook(
       userId: session.userId,
       sessionId: session.id,
       payload: {
-        recordingUrl: getRecordingUrl(ingest, session),
-        mp4Url: getRecordingUrl(ingest, session, true),
+        recordingUrl: await getRecordingUrl(ingest, session),
+        mp4Url: await getRecordingUrl(ingest, session, true),
```

Comment:
I believe this will not work anymore, since `getRecordingUrl` will return `undefined` when the recording session hasn't been created yet and it will only be created on the handling of this webhook.

Reply:
Yeah, I think you're right. But I guess we don't need these recording fields in the `recording.ready` event. Removed here: d9cd17dba40435187e79d62690731a761c2abdba